### DISCUSSION TITLE
What is the current world record for eating 🌭🌭 in 10 minutes?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#795] [Galaxy Trivia] [answered]
**Bonus:** How many lbs/kg of 🍒🥧 has this record holder eaten?

**Comment by SANKARAMDAS** [comment]
Hot dog champ does it again. Competitive eater Joey Chestnut won his record-extending 15th Nathan's Famous International Hot Dog Eating Contest, devouring an astounding 63 hot dogs and buns in 10 minutes on Monday.
59.6 lbs (27 kg) de comida em 4 horas Recorde Mundia2018 Reading Phillies Gluttony Night I break my own World Record for the most food,

**Comment by peterbaccaro** [comment, marked as answer]
The world record for eating for eating hot dogs is in 10 minutes is **76** and was achieved by Joey Chestnut (USA) in Coney Island, New York, USA, on 4 July 2021.
Joey Chestnut also has the record for eating cherry pie, **17.5lb in 8 minutes** achieved in Washington, DC on 20 July 2022.

https://majorleagueeating.com/records/

**Reply by ParanoidUser** [reply]
He must have been very hungry. 🤭
